### PR TITLE
state of the lab post

### PR DESCRIPTION
**Pull request by whoburg**
posting e-mail as blog post, as suggested by @pgkirsch.

Ready for review.

### CODE REVIEW

File: _posts/2016-01-13-state-of-the-lab.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+title: State of the Lab 2016
+author: whoburg
+---
+
+A recent e-mail I sent to the lab:
+
+<tt>
+> Folks:
+
+> Welcome back from the holidays and happy 2016. I thought last night's State of the Union address a good occasion to reflect on our first 1.4 years of existence and where we're headed.
```
Comment (pgkirsch) [inline _posts/2016-01-13-state-of-the-lab.md:11]: are you missing a "was" here, or no?

**Comment by bqpd** [comment]
Does it look best to have the email be fixed width? I wonder if just putting the text inline (and retooling / adding a line to the introduction) would be easier to read.

File: _posts/2016-01-13-state-of-the-lab.md (hunk 1 above)
Comment (whoburg) [inline _posts/2016-01-13-state-of-the-lab.md:11]: Good question.

I could be wrong, but I think this usage is correct. I'll argue for why, but please someone tell me if I'm wrong. I'm basing my opinion here on intuition only and have no actual English training to back up my claims.

I'm using a construction similar to one that's used in Robert Frost's "Stopping by Woods on a Snowy Evening", when he says "My little horse must think it queer" -- the pattern is "think [something] [adjective]" -- note he doesn't say "horse must think it is queer" or "horse must think it was queer". So, mine is "thought [last night's state of the union address] [a good occasion]". The only difference is that "a good occasion" is a noun phrase, not an adjective. But it seems similar to me.

How's that for a response you probably didn't expect to your innocent question?

Comment (pgkirsch) [inline _posts/2016-01-13-state-of-the-lab.md:11]: 10/10

Comment (bqpd) [inline _posts/2016-01-13-state-of-the-lab.md:11]: I like it as is, always a fan of removing words.

Comment (pgkirsch) [inline _posts/2016-01-13-state-of-the-lab.md:11]: In my defence, I wasn't suggesting it was incorrect. I was just making sure it was deliberate. Evidently it was.

Comment (bqpd) [inline _posts/2016-01-13-state-of-the-lab.md:11]: @pgkirsch, definitely a good question. Woody's response was just so inspiring that I found myself contributing an unnecessary two cents.

**Comment by pgkirsch** [comment]
+1 for merging once the readability/fixed width question has been addressed

**Comment by whoburg** [comment]
Ok, sounds like all comments have been addressed; someone else please press merge since I opened the PR.

**Comment by bqpd** [comment]
Would it look better unquoted?

**Comment by pgkirsch** [comment]
I almost think it would look better without the context of being an email (i.e. remove "Folks" and "See you on campus \n -Woody") but maybe that's just me.

**Comment by whoburg** [comment]
@pgkirsch, I like that suggestion. If @bqpd is on board, I'll push that change directly to master.

**Comment by bqpd** [comment]
Go for it.

**Comment by whoburg** [comment]
done. tweaked the ending too. keep the suggestions coming, if any.

**Comment by bqpd** [comment]
For a proper em-dash and not a `--`, `&em;` should do it.

**Comment by whoburg** [comment]
hmm that's not working for me. I'll just change it to a comma.

**Pull request merged**